src/main/java/au/com/sealink/quicktravel/client/models/VehicleType.java

```diff
@@ -0,0 +1,251 @@
+        this._default = _default;
+    }
+
+    public Boolean getTrailer() {
+        return trailer;
+    }
+
+    public void setTrailer(Boolean trailer) {
+        this.trailer = trailer;
+    }
+
+    public Object getMaximumLength() {
+        return maximumLength;
+    }
+
+    public void setMaximumLength(Object maximumLength) {
+        this.maximumLength = maximumLength;
+    }
+
+    public Integer getPosition() {
+        return position;
+    }
+
+    public void setPosition(Integer position) {
+        this.position = position;
+    }
+
+    public Boolean getFixedWeight() {
+        return fixedWeight;
+    }
+
+    public void setFixedWeight(Boolean fixedWeight) {
+        this.fixedWeight = fixedWeight;
+    }
+
+    public String getDefaultWeight() {
+        return defaultWeight;
+    }
+
+    public void setDefaultWeight(String defaultWeight) {
+        this.defaultWeight = defaultWeight;
+    }
+
+    public String getDetailPrompt() {
+        return detailPrompt;
+    }
+
+    public void setDetailPrompt(String detailPrompt) {
+        this.detailPrompt = detailPrompt;
+    }
+
+    public Boolean getHasCargo() {
+        return hasCargo;
+    }
+
+    public void setHasCargo(Boolean hasCargo) {
+        this.hasCargo = hasCargo;
+    }
+
+    public String getCode() {
+        return code;
+    }
+
+    public void setCode(String code) {
+        this.code = code;
+    }
+
+    public Boolean getAppearOnManifest() {
+        return appearOnManifest;
+    }
+
+    public void setAppearOnManifest(Boolean appearOnManifest) {
+        this.appearOnManifest = appearOnManifest;
+    }
+
+    public Date getCreatedAt() {
+        return createdAt;
+    }
+
+    public void setCreatedAt(Date createdAt) {
+        this.createdAt = createdAt;
+    }
+
+    public Date getUpdatedAt() {
+        return updatedAt;
+    }
+
+    public void setUpdatedAt(Date updatedAt) {
+        this.updatedAt = updatedAt;
+    }
+
+    public Boolean getMultiplyLength() {
+        return multiplyLength;
+    }
+
+    public void setMultiplyLength(Boolean multiplyLength) {
+        this.multiplyLength = multiplyLength;
+    }
+
+    public String getDefaultLength() {
+        return defaultLength;
+    }
+
+    public void setDefaultLength(String defaultLength) {
+        this.defaultLength = defaultLength;
+    }
+
+    public Boolean getFixedHeight() {
+        return fixedHeight;
+    }
+
+    public void setFixedHeight(Boolean fixedHeight) {
+        this.fixedHeight = fixedHeight;
+    }
+
+    public String getDefaultHeight() {
+        return defaultHeight;
+    }
+
+    public void setDefaultHeight(String defaultHeight) {
+        this.defaultHeight = defaultHeight;
+    }
+
+    public Boolean getRequireDropOffLocation() {
+        return requireDropOffLocation;
+    }
+
+    public void setRequireDropOffLocation(Boolean requireDropOffLocation) {
+        this.requireDropOffLocation = requireDropOffLocation;
+    }
+}
```

Comment:
Give all the "noise" of getters and setters is there any value in some kind of BIG COMMENT that says "START GETTERS AND SETTERS" and "END GETTERS AND SETTERS" so someone scrolling could easily see where the logic might be (if it exists?). Just a thought.

Reply:
getters and setters are ugly!

I have added region tags, which is supported by intellij/android studio which helps hide these blocks.